template/osfamilymap.yaml

```diff
@@ -1,5 +1,8 @@
 # -*- coding: utf-8 -*-                                                                                             
-# # vim: ft=yaml
+# vim: ft=yaml
+#
+# Setup variables using grains['os_family'] based logic.
+# Only add key:values that differ from `defaults.yaml`.
```

Comment:
Is it possible that adding all these empty `os_family`s hashes might confuse a newcomer and let her assume that they have to be listed even if not covered by the formula?

In that case, perhaps the line above can be rephrased, like 

```yaml
# You just need to add the key:values for an `os_family` that differ from `defaults.yaml`.
# Only add an `os_family` which is/will be supported by the formula (empty `os_family`s
# do not need to be listed, just added here as an example)
```

Reply:
Absolutely, just wanted to revive the useful comment that was in the `map.jinja` previously and show a list of OSes that formulas support.  I do feel like I've come across the use of `{}` before somewhere... perhaps a situation where the OS family (e.g. `Debian`) makes no changes to the defaults yet the OS derivative (e.g. `Ubuntu`) does?  Maybe just used to explicitly define the OSes?

In any case, your suggestion is excellent and I'll incorporate that.